### PR TITLE
OBPIH-799 Add quoted and current delivery date to order item table

### PR DESCRIPTION
**Pull request by DominikaHatala**

### CODE REVIEW

**Comment by DominikaHatala** [comment]
@jmiranda Could you please add database migration to this one?

File: grails-app/domain/org/pih/warehouse/order/OrderItem.groovy [hunk 1]
```diff
@@ -45,6 +45,9 @@ class OrderItem implements Serializable {
     Date dateCreated
     Date lastUpdated
 
+    Date currentDeliveryDate
```
Comment (jmiranda) [inline grails-app/domain/org/pih/warehouse/order/OrderItem.groovy]: Please change these to match the names on Shipment i.e. estimatedDeliveryDate and actualDeliveryDate.

**Comment by jmiranda** [comment]
@DominikaHatala It won't let me add you as a reviewer, but would you mind reviewing this to make sure it looks ok.

**Comment by DominikaHatala** [comment]
@jmiranda Sure, it looks ok. Will you merge it or should I do it?

**Pull request merged**